hx_lti_assignment/templates/hx_lti_assignment/create_new_assignment.html

```diff
@@ -197,7 +196,10 @@
 						<td class="field-target_object" >
 							<select id="id_assignmenttargets_set-{{forloop.counter0}}-target_object" name="assignmenttargets_set-{{forloop.counter0}}-target_object" class="selectpicker form-control" data-live-search="true">
 								{% for id,choice in form.fields.assignment_objects.choices %}
-								<option value="{{ id }}" {% if id == item.target_object.value %} selected {% endif %}>{{ choice }}</option>
+								{% target_object_is_in_course user id as exists %}
```

Comment:
Is that a custom template tag? I don't see the code for it in this PR. I agree, if it's not working as intended, then best to remove it.